manage.py

```diff
@@ -65,8 +68,27 @@ def test(path=None):
 
 @cli.command('seed_db')
 def seed_db():
```

Comment:
```suggestion
@cli.command('seed_db')
@click.pass_context
def seed_db(ctx):
```
Here is a bug fix that lets you call other cli commands (ie. import_commands) from other cli commands. Currently, I am getting an error on my end when calling `python3 manage.py seed_db` but this fixes it. See below for the rest of the fix